src/components/DataTypes/oneOf.tsx

```diff
@@ -1,20 +1,49 @@
+import { OperatorKeys } from '.';
+import { FULL_TEXT_SLUG } from '../Filters/SchemaSieve';
+import { JSONSchema } from '../Renderer/types';
 import type { CreateFilter } from './utils';
 
 export const operators = () => ({
 	is: 'is',
 	is_not: 'is not',
 });
 
-export type OperatorSlug = keyof ReturnType<typeof operators>;
+export type OperatorSlugs = OperatorKeys<typeof operators>;
 
-export const createFilter: CreateFilter<OperatorSlug> = (
+export const createFilter: CreateFilter<OperatorSlugs> = (
 	field,
 	operator,
 	value,
+	subSchema,
 ) => {
 	const operatorSlug = operator.slug;
 
-	if (operatorSlug === 'is') {
+	if (
+		operatorSlug === FULL_TEXT_SLUG &&
```

Comment:
Maybe this should move the `typeof value === 'string'` check as part of the `if` statement :thinking: ?
B/c otherwise the return is the same as for the `is` operator right below.

Reply:
yep totally right